### PR TITLE
Components4

### PR DESCRIPTION
**Pull request by tanguydamois**
Salut les amis,

I did the footer. Add it on all pages. it looks ok
I need to add the link now but don't why I was not able to find the right on. I move on the search page now.

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -6,6 +6,5 @@
   </div>
   <%= link_to "I want to find a location", places_search_path, class: "btn-home" %>
   <%= link_to "I want to start a lineup", places_map_path, class: "btn-home" %>
-<!--   <div class="footer"></div>
- --></div>
+  <%= render 'shared/footer' %>
```
Comment (lcskoerner) [inline app/views/pages/home.html.erb:9]: please add the footer to the application.html.erb instead so it will be in all pages

**Review by lcskoerner — CHANGES_REQUESTED** [review]
please see my comment. add the footer to application.html.erb so it is in all pages.

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (tanguydamois) [reply app/views/pages/home.html.erb:9]: Right! ;) it is done!

**Review by lcskoerner — CHANGES_REQUESTED** [review]
Did you remove from the other pages?

**Comment by tanguydamois** [comment]
Yes I did

Get Outlook<https://aka.ms/qtex0l> for iOS
________________________________

**Comment by lcskoerner** [comment]
I can't see the new commit with the partials removed.

**Pull request merged**